src/commons/navigationBar/subcomponents/SicpNavigationBar.tsx

```diff
@@ -1,8 +1,12 @@
-import { Alignment, Drawer, Navbar, NavbarGroup, Position } from '@blueprintjs/core';
+import { Alignment, Drawer, InputGroup, Navbar, NavbarGroup, Position } from '@blueprintjs/core';
 import { IconNames } from '@blueprintjs/icons';
+import { memoize } from 'lodash';
 import * as React from 'react';
+//import React, {useEffect} from 'react';
+//import Highlighter from 'react-highlight-words';
```

Comment:
Commented code should be removed.

Reply:
I've fixed this in #2471.

Reply:
Thank you for your amazing efforts and helps!

Reply:
I feel very sorry for the trouble caused by my incompetency; Thank you for your fixes in 2471; will definitely pay more attention to code quality in terms of type safety and readability.